server/routes.go

```diff
@@ -1073,7 +1073,7 @@ func allowedHostsMiddleware(addr net.Addr) gin.HandlerFunc {
 
 		host, _, err := net.SplitHostPort(c.Request.Host)
 		if err != nil {
-			host = c.Request.Host
+			host = strings.ToLower(c.Request.Host)
```

Comment:
Thanks for the PR and sorry for the slow review. I think this is a good change, however, should we also lowercase checks in `allowedHost`?

Reply:
yep, that also catches hostnames that aren't all lowercase, thx.